geometry_msgs/PoseStamped target_pose
duration duration # In seconds
float32 timeout # In seconds
---
uint8 SUCCEEDED = 0
uint8 FAILED = 1
uint8 CANCELLED = 2

uint8 state
trajectory_msgs/JointTrajectory trajectory
---
float32 position_error_norm
float32 orientation_error_norm